### PR TITLE
Visual sound

### PR DESCRIPTION
**Pull request by heldersepu**
Code changes for Issue #7

### CODE REVIEW

**Comment by heldersepu** [comment]
Now that we have a visual representation of the sound ...
next step will be to allow adding splits by clicking on the PictureBox and adjusting existing ones by dragging

**Comment by ProfessorCattington** [comment]
Oh I like this feature. I want to add time and file name labels over the picturebox that correspond to the labels in the DGV

**Comment by heldersepu** [comment]
Yes I like it too!
you can draw more stuff on the on the picturebox  or display a "popup" with more info

File: coldcuts/ImageForm.cs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        public bool IsBusy { get { return busy; } }
+
+        public void ShowSound(List<int> sound)
+        {
+            busy = true;
+            double max = 0;
+            int height = 100;
+            int count = sound.Count;
+            panel.Controls.Clear();
+            if (count < 90000)
```
Comment (heldersepu) [inline coldcuts/ImageForm.cs:28]: 30 minutes max

**Comment by heldersepu** [comment]
One limitation I left on place is the 30 minute max:
https://github.com/heldersepu/MP3-splitter/blob/980c8ecbd094f953d566fe30f13b130ce07b91bd/coldcuts/ImageForm.cs#L28
For bigger files we will need to create multiple PictureBoxes

**Pull request merged**